ods_ci/tests/Resources/Common.robot

```diff
@@ -200,17 +200,33 @@ Get CodeFlare Version
 #robocop: disable: line-too-long
 Wait Until Csv Is Ready
   [Documentation]   Waits ${timeout} for Operators CSV '${display_name}' to have status phase 'Succeeded'
-  [Arguments]    ${display_name}    ${timeout}=10m    ${operators_namespace}=openshift-operators
+  [Arguments]    ${display_name}    ${timeout}=10m    ${operators_namespace}=openshift-operators    ${check_interval}=5s
   Log    Waiting ${timeout} for Operator CSV '${display_name}' in ${operators_namespace} to have status phase 'Succeeded'    console=yes
   WHILE   True    limit=${timeout}
   ...    on_limit_message=${timeout} Timeout exceeded waiting for CSV '${display_name}' to be created
+    Sleep  ${check_interval}
     ${csv_created}=    Run Process    oc get csv --no-headers -n ${operators_namespace} | awk '/${display_name}/ {print \$1}'    shell=yes
-    IF    "${csv_created.stdout}" == "${EMPTY}"    CONTINUE
+    IF    not $csv_created.stdout    CONTINUE
+    Log  The Result of csv_created Output is: ${csv_created.stdout} RC: ${csv_created.rc}    console=yes
+    #  In case of upgrade there are 2 operators, we need to wait until only one will be available
+    ${lines}=     Split String    ${csv_created.stdout}    \n
+    ${line_count}=    Get Length    ${lines}
+    IF  ${line_count} > 1    CONTINUE
     ${csv_ready}=    Run Process
     ...    oc wait --timeout\=${timeout} --for jsonpath\='{.status.phase}'\=Succeeded csv -n ${operators_namespace} ${csv_created.stdout}    shell=yes
+    Log    The Result of csv_ready Output is: ${csv_ready.stdout} RC: ${csv_ready.rc}    console=yes
     IF    ${csv_ready.rc} == ${0}    BREAK
   END
 
+Check If Resource Exists By Command Output
```

Comment:
I think a better place for this keyword is ods_ci/tests/Resources/OCP.resource

Reply:
Replaced by `Search Install Plan`